[AUTHZ] Shade spark authz plugin (#5427)
apache/kyuubi: docs/security/authorization/spark/install.md

## Install

> With the `kyuubi-spark-authz_*.jar` and its transitive dependencies available or use only `kyuubi-spark-authz-bundle_*.jar` for spark runtime classpath, such as

Reviewer: ```suggestion
Use either the bundle jar `kyuubi-spark-authz-bundle_*.jar` or the `kyuubi-spark-authz_*.jar` with its transitive dependencies available for spark runtime classpath, such as
```
